flow-typed/npm/recharts_vx.x.x.js

```diff
@@ -0,0 +1,4 @@
+declare module 'recharts' {
+  declare module.exports: any;
```

Comment:
you might be able to declare classes like for the `payload` prop and stuff here if they match the recharts definition and "import" from recharts